src/config/dark-sites.config

```diff
@@ -72,6 +72,7 @@ avengedsevenfold.com
 azeria-labs.com
 azm.azerialabs.com
 badboybill.com
+based.cooking
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/147858411-48747bdb-4199-4dda-9adb-ae7b2e50659b.png)

This doesn't seem to be dark-by-default.